Makefile.win

```diff
@@ -361,6 +361,7 @@ clean:
 	-@rmdir /s/q src\mango\.venv
 	-@del /f/q src\couch\priv\couch_js\config.h
 	-@del /f/q dev\boot_node.beam dev\pbkdf2.pyc log\crash.log
+	-@del /f/q dev\erlserver.pem dev\couch_ssl_dist.conf
```

Comment:
If we're not setting dev certs on windows, let's not modify the windows makefile

Reply:
Delete